protos/peers.proto

```diff
@@ -77,11 +77,12 @@ message MongoConfig {
   string uri = 1;
   string username = 2;
   string password = 3 [(peerdb_redacted) = true];
-  bool disable_tls = 4;
-  string tls_host = 5;
-  optional string root_ca = 6 [(peerdb_redacted) = true];
-  ReadPreference read_preference = 7;
-  optional SSHConfig ssh_config = 8;
+  bool require_read_concern_majority = 4;
+  bool disable_tls = 5;
+  string tls_host = 6;
+  optional string root_ca = 7 [(peerdb_redacted) = true];
+  ReadPreference read_preference = 8;
+  optional SSHConfig ssh_config = 9;
```

Comment:
Oops, I should put the new one at the end.

Reply:
option should also be made to that false will maintain existing behavior in order to be backwards compatible